mapbox/vs30/scripts/basin_z_values/gen_sites_in_basin.py

```diff
@@ -63,26 +28,31 @@ def get_args():
         default=Path.cwd() / "sites_in_basin.csv",
     )
     arg(
-        "--remove_not_in_basin",
-        help="Remove stations not in basin",
+        "--keep_outer_basin",
+        help="Remove stations outside basins",
```

Comment:
I find this help text confusing, the description for `--keep_outer_basin` would read "Remove stations outside basins" which would imply that setting `--keep_outer_basin` would remove stations outside basins... I'd prefer to see something like "If set, keep stations outside basins". If my understanding of this flag is wrong, even more of an excuse to update the help text.